alumni/eun-plata/fizzbuzz.py

```diff
@@ -0,0 +1,31 @@
+def _is_divisible(number: int, divisor: int) -> bool:
+    """
+    Comprueba si el número es divisible
+    :param number:
+    :param divisor:
+    :return:
+    """
+    return number & divisor == 0
```

Comment:
Ojo, que eso es la operación a nivel de bit `and`.
```suggestion
    return number % divisor == 0
```